polylogue/cli/reprocess.py

```
@@ -46,6 +60,14 @@ def run_reprocess(
         data_hash = row["hash"]
         import_provider = row["provider"]
         source_path = row["source_path"] or "unknown"
+        conversation_id = row["conversation_id"] if "conversation_id" in row.keys() else None
```

Ruff (0.15.9)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)